### ISSUE TITLE
Screenshot not working

### ISSUE THREAD

**Issue by Mikaito01** [andanteyk/ElectronicObserver#34]
After the 1.2.6 update whenever I use screenshot button of ElectronicObserver, it gives me this error instead.

![image](https://cloud.githubusercontent.com/assets/8548277/7690839/dffaf5f2-fde9-11e4-82ce-a2193bd63c2e.png)

**Comment by k0286** [comment]
I have same problem, too.

After some simple testing, I think this problem is caused by using absolute path to play Kancolle.

When I traced the source code of "FormBrowser.cs" of ElectronicObserver, I found that the function, SaveScreenShot, doesn't support to capture screen with using absolute url.

Finally, I probably found solution by adding a piece of code in the SaveScreenShot function. This idea is from "[here](https://github.com/tsanie/ElectronicObserver/blob/makai/Browser/FormBrowser.cs)" at line 497-503.

    if ( viewobj == null && !isvalid( swf.DomElement as HTMLEmbed ) && !isvalid( swf.DomElement as HTMLObjectElement ) ) {
						string name = null;
						if ( swf.DomElement != null ) {
							name = swf.DomElement.GetType().FullName;
						}
						throw new InvalidOperationException( string.Format( "swf对象无效，该对象为：{0}.", name ) );
					}


I hope this can help to fix this bug. Thx a lot.
